src/import_whatsapp.js

```diff
@@ -252,13 +258,13 @@ const setImgMsgRegex = (fileType) => {
 		// console.info("ios format");
 		// iOS format
 		messageRegex =
-			/\[(\d{2}\/\d{2}\/\d{4}),\s(\d{1,2}:\d{2}:\d{2}\s(?:AM|PM))\]\s(.*?):\s(.+?)(?=\n\[|$)/gs;
+			/\[(\d{2,4}\/\d{2}\/\d{2,4}),\s(\d{1,2}:\d{2}:\d{2}\s(?:AM|PM))\]\s(.*?):\s(.+?)(?=\n\[|$)/gs;
```

Comment:
You deleted the date from the look ahead at the end.
```suggestion
			/\[(\d{2}\/\d{2}\/\d{2,4}),\s(\d{1,2}:\d{2}:\d{2}\s(?:AM|PM))\]\s(.*?):\s(.+?)(?=(\n\d{2}\/\d{2}\/\d{2,4})|$)/gs;
```

Reply:
Did you test this?

Reply:
I did and didn't notice a difference between what's on prod and my local but will put it back anyway to be safe